features/user_can_search_restaurant_assortment.feature

```diff
@@ -0,0 +1,17 @@
+Feature: visitor search for restaurant assortment
+  As a visitor
+  In order to get suggestion for a restaurant
+  I would like to be able to enter a keyword to find an assortment of restaurants to select from
+
+  Background:
+    Given the following restaurants exists
+      | name            | description                                      | assortment |
+      | ThaiTanic       | Thailands finest food, watch out for the iceberg | Thai       |
+      | Fu King Chinese | Best chinese food in town                        | Asian      |
+    And I visit the landing page
+
+  Scenario: Visitor can search for restaurants
+    And I fill in "assortment" with "Thai"
+    And I click "Search"
+    And I click "ThaiTanic"
+    Then I should be redirected to the "ThaiTanic" page
```

Comment:
We think it would be good to use the other restaurant to test that the sorting of the restaurant is functioning correctly by adding either another scenario or adding: I should not see...

Reply:
Mm, you just wrote above something conflicting with this